packaging/rpm/microshift.spec

```diff
@@ -57,6 +56,7 @@ Requires: microshift-selinux
 Requires: microshift-networking
 Requires: conntrack-tools
 Requires: sos
+Requires: greenboot
```

Comment:
MicroShift works fine without greenboot, and I don't think we want to add it as a hard requirement for the base RPM. Would it make sense to have it be a passive integration (install our script, but nothing would use it if greenboot is not installed) or to package the script in a separate RPM ("microshift-greenboot") that does depend on greenboot? The passive approach would let us use the script as a general health check tool, so I think I would prefer that if it works.

Reply:
Updated the implementation, docs and ER according to this proposal.